dimagi_lake/aggregation/aggregation_helpers/child_care_monthly.py

```diff
@@ -0,0 +1,118 @@
+from dimagi_lake.aggregation.aggregation_helpers.base_helper import BaseAggregationHelper
+from spark_session_handler import SPARK
+from consts import SERVICE_ENROLLMENT_TABLE, FLWC_LOCATION_TABLE
+
+
+class ChildCareMonthlyAggregationHelper(BaseAggregationHelper):
+
+    @property
+    def source_tablename(self):
+        return f"{self.database_name}.raw_child_care_case"
+
+    @property
+    def member_case_table(self):
+        return f"{self.database_name}.raw_member_case"
```

Comment:
why are some of these consts and some not?

Reply:
I used constants for agg tables and simple strings for the raw data tables. Will use consts for them as well